### ISSUE TITLE
Improvements & Raspbian Repo

### ISSUE THREAD

**Issue by Tqnsls** [Der-Henning/tgtg-notifications#3]
Hi Henning,
i just wanted to let you know that you maybe could add a short documentation that it might be necessary to change the uid in the docker*.yml files if one does not install via uid 1000 and gid 1000 (for exmple via root).

I also wanted to ask if you have a working raspbian repo (e.g. Raspbian Buster).
I had several errors while building the images (even manually).

Kind regards, 
Silas

**Comment by Der-Henning** [comment]
Hi Silas.
Thank you for your feedback!

Documentation is actually the biggest issue on this project. Like in most projects i guess...

I am running the service on an amd64 arch with no issues and didn't test it on arm. But i had similar issues in other projects with the node:15-alpine image. Please try building the backend and frontend images with node:15 as base image.

Kind regards,
Henning

**Comment by Tqnsls** [comment]
Hi Henning,

i just changed the node-version in {backend,frontend}/Dockerfile from "15-alpine" to "15".
When running "make images" It fails due to insufficient memory.
Can or will you assist me? im running on raspberry pi 3+ with raspbian buster, it got 1GB of RAM, I also added a swapfile but the system wont use it.
![image](https://user-images.githubusercontent.com/67681686/130240382-f1063125-e66e-47bf-b701-d09d1ec5e1f9.png)

I thought that i could execute the commands manually and maybe check where and why exactly it fails.
![image](https://user-images.githubusercontent.com/67681686/130239230-e38ef0f1-645c-49db-a501-5fff0f2a33c5.png)
Can it even work that way?

**Comment by Der-Henning** [comment]
Hi Silas.

1GB is apparently not enough memory to build the react frontend. With buildx it is possible to build multi arch images on a different machine. For example you can use github workflows to build these images and push them to your registry.
Feel free to fork and create a pull request.

Here is an example workflow with exactly that purpose from on of my projects:
[docker-multi-arch.yml](https://github.com/Der-Henning/mopsy-react/blob/master/.github/workflows/docker-multi-arch.yml)
This workflow creates linux images for amd64, arm/v7 and arm64 which should run on rpi3 and rpi4.
It is set up to push the images to the hub.docker.com repository specified with the secrets DOCKER_USERNAME and DOCKER_PASSWORD.

**Comment by Tqnsls** [comment]
Hi Henning,
after trying to set tgtg-notifications up on a virtualized CentOS with several errors due to dependencies, the last state was that I could "make images" and "make start" but afterwards the frontend and backend kept starting and stopping all over.
Then I finally could set it up on a fresh ubuntu 20.04.
Now my question is, which IFTTT-event does "my" webservice use? I guess its the same as the official ("tgtg_notification"), right?

When registering a new user on my webservice and trying to add a webhook importing the favourite stores it kept saying "Authentication Failed", don't know which authentication exactly failed, the connection to the mongodb works, users are created..

**Comment by Der-Henning** [comment]
Yes "tgtg_notifications". I am planning to implement webhooks in general. So users can connect any service with webhooks, like smart home or other notification services.

At the moment only TGTG accounts registered with mail and password are supported. Accounts with login via Apple etc. are not working.
You need one fresh tgtg account for the service to store the favorites of all users. The credentials musst be saved in tgtg.env.

**Comment by Tqnsls** [comment]
Hi Henning,
i set up a new tgtg-master user to store all favorites in it.
I logged in tgt-notifications available at http://<private-IP>:4000/account as tgtg-debug and tried to add my IFTTT-key, but there it says "Authentication Failed".
![image](https://user-images.githubusercontent.com/67681686/130784540-4544d7f8-89ca-4db2-81fb-3b1c6b7b83ba.png)

Also when trying to import the favorites from tgtg-debug it says the same.
![image](https://user-images.githubusercontent.com/67681686/130784706-c4b750b3-4fca-4d92-8678-b8713b60f4e3.png)
The login of the tgtg-debug User in the offical tgtg-app definetly works.

Also when I try to logout at http://192.168.2.60:4000 it says "Authentication Failed", kinda weird.. :D

**Comment by Tqnsls** [comment]
Hi Henning,
I just tested in a browser with deleted cookies, cache etc. but unfortunately the same error.
